src/Evolution/Executables/GeneralizedHarmonic/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@
 set(LIBS_TO_LINK
   Amr
   AmrCriteria
+  AmrPolicies
```

Comment:
Add `AmrPolicies` as a dependency to `ParallelAmr`. You shouldn't have to add it to each executable because the executable only includes it through `ParallelAmr`.